unity-client/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/Gif/GifProcessor/GifProcessor.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System.Collections;
+using System;
+using DCL.Helpers;
+using UnityEngine.Networking;
+
+/// <summary>
+/// GifProcessor: Is in charge of choosing which gif processor tu use (typescript's webworker through GIFProcessingBridge or Unity's plugin UniGif)
+/// for downloading, processing and discarding gifs
+/// </summary>
+public class GifProcessor
+{
+    private bool jsGIFProcessingEnabled = false;
```

Comment:
Is it OK that this is false by default?

Reply:
Yeah, It shouldn't be a problem since Gif promises are instanced after the engine is initialized, so the KernelConfig (that set that boolean) is sent to unity before any promise is created